demo-code/seed_data.json

```diff
@@ -84,7 +84,7 @@
         "pk": 1,
         "fields": {
             "name": "The Sun Tour",
-            "description": "pdfs",
+            "description": "A short tour of the sun plus the planet that relies on it for life.",
```

Comment:
"pdfs" is a weird description, looks like a bug, so I added actual descriptions

Reply:
I was talking to @tonybaloney about adding witty/educational descriptions in the data as well 👍🏾